runtime/src/lib.rs

```diff
@@ -772,6 +772,20 @@ parameter_types! {
 	pub const MaxEncodedLength: u32 = 15_360;
 }
 
+impl pallet_scoring::Config for Runtime {
+	type RatingCreatorIdOf = DidIdentifier;
+	type EnsureOrigin = pallet_did::EnsureDidOrigin<DidIdentifier, AccountId>;
+	type OriginSuccess = pallet_did::DidRawOrigin<AccountId, DidIdentifier>;
+	type RuntimeEvent = RuntimeEvent;
+	type MinScoreValue = MinScoreValue;
+	type WeightInfo = weights::pallet_scoring::WeightInfo<Runtime>;
+	type ValueLimit = ConstU32<72>;
```

Comment:
where is this used?

Reply:
pub type TransactionIdentifierOf<T> = BoundedVec<u8, <T as Config>::ValueLimit>;